transport/rif_comms/client.py

```diff
@@ -92,8 +84,7 @@ def unsubscribe_from(self, rsk_address: Address):
         Invokes the CloseTopic GRPC API endpoint.
         :param rsk_address: RSK address which corresponds to the topic which the client is unsubscribing from.
         """
-        topic_id = self._get_peer_id(to_checksum_address(rsk_address))
-        self.stub.CloseTopic(Channel(channelId=topic_id))
+        self.stub.CloseTopicWithRskAddress(RskAddress(address=to_checksum_address(rsk_address)))
```

Comment:
i found confusing that here we use to_checksum_address and on SendMessageToRskAddress we not

i think we must use always the checksum

Reply:
agreed

Reply:
fixed